### ISSUE TITLE
Links in disclosure card content still focusable when disclosure card is collapsed

### ISSUE THREAD

**Issue by richardfxr** [richardfxr/sity#2]
## Expected Behavior

In-line links in the disclosure card content cannot be focused (using the tab key) if the disclosure card is collapsed. Disclosure cards can be found in the [FAQ](https://sity-earth--dev-vc9f6b05.web.app/faq) and [updates](https://sity-earth--dev-vc9f6b05.web.app/updates) page.

## Current Behavior

In-line links can still be focused. While the disclosure card opens, the "aria-hidden" attribute is still set to "true," resulting in screen readers not being able to read the link.

## Version

v0.1.13 in Chrome.

## Possible Solution

Add dynamic "tabindex" attribute to avoid focusing on these links
